modules/mono/glue/GodotSharp/GodotSharp/Core/Colors.cs

```diff
@@ -9,7 +9,7 @@ namespace Godot
     /// </summary>
     public static class Colors
     {
-        // Color names and values are derived from core/color_names.inc
+        // Color names and values are derived from core/math/color_names.inc
         internal static readonly Dictionary<string, Color> namedColors = new Dictionary<string, Color> {
             {"aliceblue", new Color(0.94f, 0.97f, 1.00f)},
```

Comment:
Since the core names are uppercase, I think this backing dictionary should be too. We can replace `.ToLower()` with `.ToUpper()` in `ColorN` to be more similar to core, but I think we should keep the lack of underscores in the C# dictionary.

Reply:
Done.